minecraft_atomizer.py

```diff
@@ -40,30 +33,25 @@
 def setup_args():
     '''
     Sets up command-line arguments
-
     If running from an IDE instead of the command line, simply set these
     values here (e.g., ITEM_NAME = 'torch', WRITE = TRUE, etc.)
-
     Attributes
     ----------
     ITEM_NAME : string (optional, default='torch')
         The name of the item to be atomized
-
     WRITE : bool (optional, default=False)
         If True, write the results of the atmoization to a file called
         output.txt
-
     DEBUG : bool (optional, default=False)
         if True, print out useful debug information
-
     parser : ArgumentParser
         Parses command-line arguments
-
     args : Parsed Arguments
     '''
     global ITEM_NAME
     global WRITE
     global DEBUG
+    global REDUCED
```

Comment:
Please change the name of this bool, the word reduced may be confusing, especially in this script, which 'reduces' items to their base components. My suggetion is HUMAN_READABLE (the -h making output easier to read is also common among *nix programs).